pkg/cmd/pulumi/convert.go

```diff
@@ -46,11 +49,11 @@ func newConvertCmd() *cobra.Command {
 
 			var programGenerator programGeneratorFunc
 			switch language {
-			case langDotnet:
+			case "c#":
 				programGenerator = dotnet.GenerateProgram
 			case langGo:
 				programGenerator = gogen.GenerateProgram
-			case langNodejs:
```

Comment:
We're a little inconsistent about if we really mean languages or runtimes 😞 
go and python are easy, langauge == runtime. But nodejs will run javascript or typescript but it codegens typescript, and dotnet will run fsharp, csharp, or vb (or anything else dotnet run will run!) but codegens csharp.
Worth giving some thoughts about this going forward...

Reply:
Definitely. At one point I also had in the file "cs", "ts", and "py" as well. What do you think?

Reply:
Probably not needed for now. Not even sure if we even really need "c#", one name for each thing will probably suffice.